LiteDB.FSharp/Json.fs

```diff
@@ -80,6 +80,8 @@ type MapSerializer<'k,'v when 'k : comparison>() =
 
 module private Cache =
 
+    let customSerializer = Dictionary<Type, obj -> BsonDocument>()
```

Comment:
(obj -> BsonDocument) but not (obj -> BsonValue)

Reply:
Maybe because `BsonDocument` is a `BsonValue`? I am not sure though